game/src/graphic/hud/PauseMenu.java

```diff
@@ -18,22 +24,64 @@ public PauseMenu() {
     /** Creates a new PauseMenu with a given Spritebatch */
     public PauseMenu(SpriteBatch batch) {
         super(batch);
-        ScreenText screenText =
-                new ScreenText(
-                        "Paused",
-                        new Point(0, 0),
-                        3,
-                        new LabelStyleBuilder(FontBuilder.DEFAULT_FONT)
-                                .setFontcolor(Color.RED)
-                                .build());
-        screenText.setFontScale(3);
-        screenText.setPosition(
-                (Constants.WINDOW_WIDTH) / 2f - screenText.getWidth(),
-                (Constants.WINDOW_HEIGHT) / 1.5f + screenText.getHeight(),
-                Align.center | Align.bottom);
-        add((T) screenText);
+    /**pauseMsg is only used for testing the function showInfoText().
+    * ToDo: Transfer content from external source via parameter!!! */
+        final String pauseMsg = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.";
+        showInfoText( pauseMsg );
+        dialog.SetResult(true);
+        add((T)dialog );
         hideMenu();
     }
+/**function showInfoText to pause the game, display the content in the GUI, after pressing the button "End Pause" the game is continued.*/
+    static void showInfoText(String msg)
```

Comment:
Das hier hätte ich eigentlich gerne als Einstiegspunkt.
Die Methode könnte in einer neuen Klasse `UITools` static sein.
Als User möchte ich nur irgendwo im Code sagen "showInfoText("Mein toller Text")" und um mehr will ich mich nicht kümmern. 
Die Methode legt mir dir dann das `PausMenue` an und schließt es auch wieder, wenn der Button gedrückt wird. 

Edit: Vermutlich ist der Name "PausMenue" auch nicht mehr so gut. Das verbindet man das Gedanklich sofort mit der Spielpause. Eigentlich braucht es ja nur ein Menü, welches Text angezeigt und zusätzlich halt das Spiel pausiert.